src/main/scala/is/hail/variant/GenomeReference.scala

```diff
@@ -200,8 +203,9 @@ object GenomeReference {
   var defaultReference = GRCh37
 
   def addReference(gr: GenomeReference) {
-    if (references.contains(gr.name))
-      fatal(s"Cannot add reference genome. Reference genome `${ gr.name }' already exists.")
+    if (hasReference(gr.name))
+      fatal(s"Reference genome `${ gr.name }' already exists.")
```

Comment:
I think it'd be helpful to print out the names of the current keys, in case users really mean to use one that's already there.